### DISCUSSION TITLE
[FIRMWARE REQUEST] Multiple inputs from single GPIO

### DISCUSSION THREAD

**Discussion by Fabianobr79** [OpenStickCommunity/GP2040-CE#449] [Q&A]
criar um botão  igual a combinação de dois botões: um botão pressionado mandar um input de dois botões combinados. algo parecido com o hitbox do Daigo com 11 `botões` de ataques

**Comment by TheTrainGoes** [comment]
Hi @Fabianobr79 ,

This is already being worked on.  I will close this.
